belvedere/src/main/java/zendesk/belvedere/BelvedereDialog.java

```diff
@@ -232,7 +232,7 @@ public static AttachmentSource from(MediaIntent belvedereIntent, Context context
             } else if (belvedereIntent.getTarget() == MediaIntent.TARGET_DOCUMENT) {
                 return new AttachmentSource(R.drawable.belvedere_ic_image, context.getString(R.string.belvedere_dialog_gallery));
             } else {
-                return new AttachmentSource(-1, context.getString(R.string.belvedere_dialog_unknown));
+                return new AttachmentSource(-1, "");
```

Comment:
?

Reply:
This was an edge case in the deprecated dialog from v1, we don't need a translated string here.